packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -41,6 +59,7 @@
               :is-folder-share="highlightedFile.isFolder"
               :link="link"
               :modifiable="canEdit"
+              :can-rename="true"
```

Comment:
is renaming still possible even when `canEdit` (see one line above) evaluates to `false`? 🤔 For the quicklinks I get that they have a hardcoded `false` here, but for the other links the permissions should still be respected I think.

Reply:
all good. keep it like it is for now...